### PR TITLE
Update history.rst with changes from #68

### PR DESCRIPTION
**Pull request by nbren12**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -6,12 +6,19 @@ latest
...
 
 Minor changes:
-
+~~~~~~~~~~~~~
 * Added the flag ``--mca btl_vader_single_copy_mechanism none to mpirun in fv3run`` to mpirun in fv3run
 * Add ReadTheDocs configuration file
 * Do not require output dir and fv3config to be remote in ``run_kubernetes``
 * Fix bug when submitting k8s jobs with images that have an "_" in them
 
+Breaking changes
```
Comment (mcgibbon) [inline HISTORY.rst]: We've been putting these under "Major changes" in fv3gfs-python, but I don't object to this heading.

File: HISTORY.rst [hunk 2]
```diff
@@ -6,12 +6,19 @@ latest
...
+~~~~~~~~~~~~~~~~
+* Refactored run_kuberentes and run_docker to call run_native via serializing
```
Comment (mcgibbon) [inline HISTORY.rst]: `run_kubernetes`

File: HISTORY.rst [hunk 3]
```diff
@@ -6,12 +6,19 @@ latest
...
+  their args/kwargs as json strings. While no API changes occured, the
```
Comment (mcgibbon) [inline HISTORY.rst]: API changes did occur, even if the run_native api change was backwards-compatible. A new API was introduced for run_native, and run_kubernetes and run_docker were modified to rely on the new API. Could add "via a new API serializing" above and remove "While no API changes occurred" here.

File: HISTORY.rst (hunk 2 above)
Comment (nbren12) [reply HISTORY.rst]: done

Comment (mcgibbon) [inline HISTORY.rst]:
```suggestion
* Refactored run_kubernetes and run_docker to call run_native via a new API serializing
```

File: HISTORY.rst (hunk 3 above)
Comment (mcgibbon) [inline HISTORY.rst]:
```suggestion
  their args/kwargs as json strings. The
```

**Pull request merged**